modules/ROOT/pages/bolt/message.adoc

```diff
@@ -279,8 +279,9 @@ There are three different kinds of messages:
 | <<messages-hello, `HELLO`>>
 | `01`
 | Request
-| `extra::Dictionary(user_agent::String, routing::Dictionary(address::String))`
+| `extra::Dictionary(user_agent::String, notifications_minimum_severity::String, notifications_disabled_categories::List<String>, routing::Dictionary(address::String))`
```

Comment:
I don't think it makes any difference code-wise, but I would put the new ones _after_ `routing`, just to further clarify the chronological order in which they appeared.